### PR TITLE
Light Hotspots -ne Data Only HotSpots

### PR DESCRIPTION
**Pull request by jkowalcz**
Moved all HotSpot variant docs into mine-hnt. Updated all links to point to the "correct" Light Spot or Data Only Hot spot locations.  Updated data-only-hotspots to remove all extra "light hotspot" references and amde them "Data only References"

**One unknown is the side bar Menu, I'd like someone to look at that in detail.**

### CODE REVIEW

**Comment by jkowalcz** [comment]
Reviewing broken links build -- Reviewed links and they are all there.  I believe that this is due to the new file locations and the build process not having them in order? Any help appreciated.

File: docs/mine-hnt/data-only-guides/dragino.mdx [hunk 1]
```diff
@@ -160,15 +160,15 @@ packet forwarder is communicating with the Helium Gateway service running on the
...
 LoRaWAN settings are not correct per the above.
 
 ## COMING SOON: Onboard as Data-Only Hotspot
 
 As part of Milestone 1.5 of the Light Gateway project roadmap, you will be able to
-onboard light gateways such as the Dragino to appear on the coverage map, and then,
+onboard data only hotspots such as the Dragino to appear on the coverage map, and then,
 in Milestone 2, earn HNT for data transfers (but not for Proof of Coverage). See
-[Light Gateways - BETA](https://docs.helium.com/mine-hnt/light-hotspots) for
+[Light HotSpots - BETA](/mine-hnt/light-hotspots) for
```
Comment (jthiller) [inline docs/mine-hnt/data-only-guides/dragino.mdx]: Lowercase in "Hotspots"
```suggestion
[Light Hotspots - BETA](/mine-hnt/light-hotspots) for
```

Comment (jkowalcz) [reply docs/mine-hnt/data-only-guides/dragino.mdx]: I fixed this, also the Only place in the whole tree like that! Doh!

**Review by kent-williams — COMMENTED** [review]
@jkowalcz You need to update the [slug](https://github.com/helium/docs/pull/581/files#diff-9a3972987fa80cd6da68042d1700c0728129bb0b535542cd598224c2e54002beR5) after changing the filename and location.

**Comment by kent-williams** [comment]
Please don't modify the .gitignore and instead remove any incorrect files you are creating locally.

**Comment by jkowalcz** [comment]
> Please don't modify the .gitignore and instead remove any incorrect files you are creating locally.

OK, Noob Yarn developer Sorry.  When I installed Yarn there were a bunch of files created on the windows install. I'll revert the PR then and try again.

File: docs/mine-hnt/data-only-hotspots.mdx [hunk 2]
```diff
@@ -132,3 +132,20 @@ transaction hash at the following endpoint: `https://api.helium.io/v1/pending_tr
 
 Once the transaction clears, your Data Only Hotspot is ready to start
 earning!
+
+## Data-Only Hotspot Setup Guides
+
+- [Dragino LPS80/DLOS8](/mine-hnt/data-only-guides/dragino)
+  - Draginos will only mine HNT as a Data Only Hotspot
+- [Kerlink gateways]((/mine-hnt/data-only-guides/kerlink)
```
Comment (PharkMillups) [inline docs/mine-hnt/data-only-hotspots.mdx]:
```suggestion
- [Kerlink gateways](/mine-hnt/data-only-guides/kerlink)
```

File: docs/mine-hnt/data-only-hotspots.mdx [hunk 3]
```diff
@@ -132,3 +132,20 @@ transaction hash at the following endpoint: `https://api.helium.io/v1/pending_tr
...
+  - Kerlink gateways will only mine HNT as Data Only Hotspot unless it is
+    specified as a Helium-Compatible Kerlink Miner.
+- [RAK Concentrator (RAK2245/RAK2247/RAK2287) + Raspberry Pi 2, 3, or 4]((/mine-hnt/data-only-guides/rak-concentrators)
```
Comment (PharkMillups) [inline docs/mine-hnt/data-only-hotspots.mdx]:
```suggestion
- [RAK Concentrator (RAK2245/RAK2247/RAK2287) + Raspberry Pi 2, 3, or 4](mine-hnt/data-only-guides/rak-concentrators)
```

**Review by PharkMillups — DISMISSED** [review]

**Comment by jkowalcz** [comment]
Restarting this one as it's a complex git hole...

**Pull request closed without merging**